setup.py

```diff
@@ -25,7 +25,7 @@
         "snorkel-metal>=0.3.3, <0.4.0",
         "spacy>=2.0.12, <3.0.0",
         "sqlalchemy[postgresql_psycopg2binary]>=1.2.12, <2.0.0",
-        "torch>=0.4.1, <0.5.0",
+        "torch==0.4.1.post2",
```

Comment:
```suggestion
        "torch>=0.4.1.post2, <0.5.0",
```
I believe we should still be able to have an upper limit on the dependency, like this.